### PR TITLE
Address review comments

### PR DESCRIPTION
**Pull request by dhardy**

### CODE REVIEW

File: src/guide-gen.md [hunk 1]
```diff
@@ -18,17 +18,16 @@ A **true** random number generator (TRNG) is something which produces random
...
-Since from our point of view these things are unpredictable and we require some
-source of *entropy*, we are forced to assume that at least some such processes
-are truely random.)
+For our purposes, it is sufficient that they are not distinguishable from true
+randomness.)
 
 Note that these processes are often biased, thus some type of *debiasing* must
 be used to yield the unbiased random data we desire.
 
 ## Psuedo-random number generators
```
Comment (vks) [inline src/guide-gen.md]: You missed the typo in the heading.

File: src/guide-gen.md [hunk 2]
```diff
@@ -88,16 +87,43 @@ Some CSPRNGs additionally satisfy a third property:
...
+An HRNG may be used to provide the seed for a PRNG, although usually this is not
+the only way to obtain a secure seed (see the next section). An HRNG might
+replace a PRNG altogether, although since we now have very fast and very strong
+software PRNGs, and since software implementations are easier to verify than
+hardware ones, this is often not the preferred solution.
+
+Since a PRNG needs a random seed value to be secure, an HRNG may be used to
+provide that seed, or even replace the need for a PRNG. However, since the goal
+is usually "only" to produce unpredictable random values, it is not necessary to
+depend on a *true* random number generator; instead, one can usually
```
Comment (vks) [inline src/guide-gen.md]: Something is missing here?

File: src/guide-gen.md [hunk 3]
```diff
@@ -88,16 +87,43 @@ Some CSPRNGs additionally satisfy a third property:
...
 
+Ideally, a random boolean or a coin flip has 1 bit of entropy, although if the
+value is biased, there will be less. Shannon Entropy attempts to measure this.
+
+For example, a Unix timestamp (seconds since the start of 1970) contains both
+high- and low-resolution data. This is typically a 32-bit number, but the amount
+of *entropy* will depend on how precisely a hypothetical attacker can guess the
+number. If an attacker can guess the number to the nearest minute, this may be
+approximately 6 bits (2^6 = 64); if an attacker can guess this to the second,
+this is 0 bits. [`JitterRng`] uses this concept to scavange entropy without a
```
Comment (vks) [inline src/guide-gen.md]: scavange -> scavenge

Comment (vks) [reply src/guide-gen.md]: Shouldn't it be "an HRNG" if you spell the acronym?

**Comment by dhardy** [comment]
Sorry, forgot about this.

My spell-checker found a few more mistakes... and quite a few false-positives, including 'and', 'our'.

**Pull request merged**